corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_module_overwrite.html

```diff
@@ -4,17 +4,49 @@
 <form class="form-inline" method='POST' action='{% url "overwrite_module_case_list" domain app.id module.unique_id %}'>
   {% csrf_token %}
   <fieldset>
-    <label for="other_module_id">{% trans "Module" %}: </label>
-    <select name='source_module_unique_id' class='form-control'>{% for other_mod in available_modules %}
-      <option value={{ other_mod.unique_id }}>{{ other_mod.name|html_trans:langs }}</option>
-    {% endfor %}</select>
-    <input name="detail_type" type="hidden" value="{{ detail_type }}" />
-    <button class='btn btn-danger' type="submit">
-      <i class="fa fa-copy"></i>
-      {% if detail_type == 'short' %}
-        {% trans "Overwrite Case List" %}
-      {% else %}
-        {% trans "Overwrite Case Detail" %}
-      {% endif %}
+       <div class="contaianer-fluid">
```

Comment:
`contaianer-fluid` is misspelled, so that class isn't doing anything - the UI looks good, so you can probably just remove it.